spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
@RX14 you are probably right and things might be as they are because of evolution only and could be changed. There is no point in using bundled stdlib with compiler with current master code.